### PR TITLE
Susy 2lss maysync

### PR DESCRIPTION
**Pull request by cbotta**
@mmarionncern @gpetruc @peruzzim
all this, which includes the commits on the ReCleaner made by Matthieu yesterday, corresponds to the version of the 2lss synch numbers (table and list) I'm updating right now in the ra5 twiki. 

@mmarionncern 
could you please check if using this new ReCleaner you'll get the same evt list as mine?

cristina

### CODE REVIEW

File: CMGTools/TTHAnalysis/python/tools/leptonJetReCleaner.py [hunk 1]
```diff
@@ -152,8 +225,10 @@ def minMllTL(self, lepsl, lepst, bothcut=lambda lep:True, onecut=lambda lep:True
             if len(pairs):
                 return min(pairs)
             return -1
-    def bestSSPair(self,leps,byflav,cut=lambda lep:True):
+    def bestSSPair(self,leps,byflav,bypassMV,cut=lambda lep:True):
         ret = (0,1)
+        if len(leps) < 2:
+            ret = (0,1)
```
Comment (gpetruc) [inline CMGTools/TTHAnalysis/python/tools/leptonJetReCleaner.py:231]: It's already set to 0,1 the line before

**Comment by gpetruc** [comment]
Should we also add iL1V and iL2V (or iL1Vp  iL2Vp) with loose leptons, for events in the FR application region? And similarly a SR definition based on those leptons? (maybe by making the existing blob of code a function taking l1,l2,ht,nj,nb ?) 
Anyway, it can be done in another PR

File: CMGTools/TTHAnalysis/python/tools/leptonJetReCleaner.py (hunk 1 above)
Comment (cbotta) [inline CMGTools/TTHAnalysis/python/tools/leptonJetReCleaner.py:231]: I'll remove this

**Comment by cbotta** [comment]
Yes, agreed. I'll add also these indexes and the SR with these leptons. I'll do it in a next PR where I will also add the remaining SR (for hl, ll), making the SR function dependent also on the lepPt

**Pull request merged**